salt/metalk8s/addons/dex/deployed/service-configuration.sls

```diff
@@ -23,18 +23,23 @@ Create dex-config ConfigMap:
           config.yaml: |-
             apiVersion: addons.metalk8s.scality.com
             kind: DexConfig
-            spec:
-              deployment:
-                replicas: 2
-              localuserstore:
-                enabled: true
-                userlist:
-                  - email: "admin@metalk8s.invalid"
-                    hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
-                    username: "admin"
-                    userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
-              connectors: []
-
+            spec: {}
+        # example configuration
+          # example.yaml: |-
+          #   apiVersion: addons.metalk8s.scality.com
+          #   kind: DexConfig
+          #   spec:
+          #     # Configure the Dex Deployment
+          #     deployment:
+          #       replicas: 2
+          #     localuserstore:
+          #       enabled: true
+          #       userlist:
+          #       - email: "read_only@metalk8s.invalid"
+          #         hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
+          #         username: "read_only"
+          #         userID: "89b3684b-db28-b7v3-a99c-j99305v2"
+          #     connectors: []
```

Comment:
Not sure we want to keep this here as it does not really help, we should rather have some documentation exposing the CSC defaults.